pkgs/tools/X11/wpgtk/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, python3, python36Packages, fetchFromGitHub, pywal, feh, libxslt, imagemagick,
+  gobjectIntrospection, gtk3, wrapGAppsHook, gnome3 }:
+
+python36Packages.buildPythonApplication rec {
+  name = "${pname}-${version}";
```

Comment:
This line can be removed. Packages built with `buildPythonApplication` do this by default.